packaging/windows/internal/cuda_install.bat

```diff
@@ -90,6 +96,15 @@ if not exist "%SRC_DIR%\temp_build\cudnn-10.2-windows10-x64-v7.6.5.32.zip" (
     set "CUDNN_SETUP_FILE=%SRC_DIR%\temp_build\cudnn-10.2-windows10-x64-v7.6.5.32.zip"
 )
 
+rem I tested, it's only necessary for cu102.
+if not exist "%SRC_DIR%\temp_build\gpu_driver_dlls.7z" (
+    curl -k -L "https://drive.google.com/u/0/uc?id=1injUyo3lnarMgWyRcXqKg4UGnN0ysmuq&export=download" --output "%SRC_DIR%\temp_build\gpu_driver_dlls.zip"
+    if errorlevel 1 exit /b 1
+)
+
+echo Installing GPU driver DLLs
+7z x %SRC_DIR%\temp_build\gpu_driver_dlls.zip -aoa -o"C:\Windows\System32"
+
```

Comment:
Do you know, why those extra dlls are not need/missing them for CUDA builds/tests, but are needed for vision?

Reply:
@malfet , Do you mean the CUDA builds/tests in pytorch?
I think it was just copied from builder.
https://github.com/pytorch/builder/blob/17307c3a1d3f31465dcf07b4c2646ffcfee2bff8/windows/internal/cuda_install.bat#L224-L225

according to peter's comments, it might be out of date.
https://github.com/pytorch/vision/pull/4058#issuecomment-863373493
![image](https://user-images.githubusercontent.com/16190118/124062662-aaf91480-da63-11eb-99cc-931256b4a00a.png)

But the difference is that Pytorch build is in  CPU image but visioin build/test are both in GPU image.
In circleci GPU image, the default cuda is 10.1. while installing cuda10.2,  if we don't copy the gpu_drivers_dll, the nvidia-smi still show the cuda versions is 10.1. And it will show cuda runtime isn't found in the log.

Reply:
https://app.circleci.com/pipelines/github/pytorch/vision/9359/workflows/602ca040-f53f-4001-9ba6-4f8385b911ea/jobs/687596
https://circleci.com/api/v1.1/project/github/pytorch/vision/687596/output/104/0?file=true&allocation-id=60dd3c3c26edbe5ec6040904-0-build%2F7410C0A2
![image](https://user-images.githubusercontent.com/16190118/124063831-4ee3bf80-da66-11eb-88f6-3077a24b8ee1.png)

Reply:
@mszhanyi 
```
UserWarning: CUDA initialization: The NVIDIA driver on your system is too old (found version 10010). Please update your GPU driver by downloading and installing a new version from the URL: http://www.nvidia.com/Download/index.aspx Alternatively, go to: https://pytorch.org to install a PyTorch version that has been compiled with your version of the CUDA driver. (Triggered internally at  ..\c10\cuda\CUDAFunctions.cpp:115.)
  return torch._C._cuda_getDeviceCount() > 0
```
This is only a warning which won't block the builds. The actual error is the following one. Looks like the env activation somehow fails.
```
UserWarning: Error checking compiler version for cl: [WinError 2] The system cannot find the file specified
  warnings.warn(f'Error checking compiler version for {compiler}: {error}')
```